addons/rose/addon/components/rose/form/fieldset/index.hbs

```diff
@@ -0,0 +1,20 @@
+<fieldset ...attributes class='rose-form-fieldset'>
+    <legend 
+        id={{concat 'title-' this.id}}
+        aria-describedby={{concat 'description-' this.id}}
+        class='rose-form-fieldset-title'
+    >
+        {{yield to='title'}}
+    </legend>
+    <div
```

Comment:
Will everything render as expected if a description is not passed?

Reply:
Yes. I just tested with this code as render no problem:
```javascript
<Rose::Form::Fieldset>
  <:title>Title</:title>
  <:body>Rest of the form</:body>
</Rose::Form::Fieldset>
```

Reply:
Yes I think this will be important to handle because of the `aria-describedby`.  It may be a poor experience for some users if they rely on that attribute and there is no associated text.

Reply:
That is right. Even I am deleting the `aria-describedby` as per your suggestion, I agree if does not receive description to not render it.